QuizSolver/QuizSolver/Model/Quiz.cs

```diff
@@ -23,12 +23,18 @@ public Quiz()
 			questions = new ObservableCollection<Question>();
 		}
 
-        public Quiz(String name, ObservableCollection<Question> questions)
+		public Quiz(String name, ObservableCollection<Question> questions)
 		{
 			Name = name;
 			Questions = questions;
 		}
 
+		public Quiz Copy()
```

Comment:
Jak tutaj tworzysz kopie quizu, to przekazujesz to konstruktora `Questions`, czyli referencje to tego `ObservableCollection`, która jest w oryginale. Czyli jeśli w kopii coś zmienisz, to odwołujesz się do `Questions` z oryginału.

Reply:
🤓

Reply:
🦜